sdk/communication/azure-communication-chat/src/main/java/com/azure/communication/chat/models/ChatMessage.java

```diff
@@ -21,14 +21,9 @@
     /*
      * Type of the chat message.
      *
-     * Possible values:
-     * - Text
-     * - ThreadActivity/TopicUpdate
-     * - ThreadActivity/AddMember
-     * - ThreadActivity/DeleteMember
      */
     @JsonProperty(value = "type")
-    private String type;
+    private ChatMessageType type;
```

Comment:
Did you forget about "required = true", or did you leave it out on purpose?

Reply:
Same for other properties

Reply:
I didn't know about it, is there a release note for the api that covers this?